<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntHouse GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>805</width>
     <height>507</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="dictionary">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>96</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Dictionary</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>2</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row/>
            <row/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ressources">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>447</width>
           <height>143</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Ressources</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="segmLabel">
          <property name="text">
           <string>Segm.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="segmCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>View</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="contentCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>content</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x_shift">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y_shift">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="z_shift">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="segmLoadButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Load...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Skeleton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="skelCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="contentSkelCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x_shift_skel">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y_shift_skel">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="z_shift_skel">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skelLoadButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="axisSelection"/>
            </item>
            <item>
             <widget class="QLabel" name="SlicePosition">
              <property name="text">
               <string>Position</string>
              </property>
              <property name="buddy">
               <cstring>sequenceSlider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sequenceSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Intensity</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>scrollArea</cstring>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QSpinBox" name="spinMinIntensity"/>
              </item>
              <item>
               <widget class="QSpinBox" name="spinMaxIntensity"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Threshold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="binSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="binPushButton">
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="text">
           <string>Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>315</width>
             <height>351</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="_labelSliceView">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_project"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="actionClose_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_folder">
   <property name="text">
    <string>Import dicom folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionClose_folder">
   <property name="text">
    <string>Close folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
